tmk_core/common/keyboard.c

```diff
@@ -64,20 +64,41 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 #ifdef MATRIX_HAS_GHOST
+static uint16_t matrix_ghost_check[MATRIX_ROWS];
```

Comment:
I think the ARM has function alignment on, so that can make the size a bit hard to analyze.